mizar/daemon/interface_service.py

```diff
@@ -34,6 +34,7 @@ def __init__(self):
         self.iproute = IPRoute()
         self.interfaces = {}  # In-memory tracking for Pod interfaces
         self.queued_pods = set()  # A set of pods, with queued interfaces (to be consumed)
+        self.dequeued_pods = set()  # A set of pods, with queued interfaces (to be deleted)
```

Comment:
Do we actually need this? In the create we need a kind of synchronization between the operator and the cni, that's why we have the queue. But for delete, you can simply delete the interface in the CNI and delete the simple endpoint in the operator.